main.tex

```diff
@@ -1,23 +1,23 @@
-\documentclass[titlepage]{book}
-\usepackage[utf8]{inputenc}
-
+\documentclass[letterpaper,american,final]{memoir}
 \include{preamble}
-
+\title{Understanding Analysis Solutions}
+\author{Michael You}
+\date{}
```

Comment:
I honestly can't say this is necessary either. I have a preference to keep these things in the preamble